### PR TITLE
styling on cust register, login pages, handyman views and js

### PR DESCRIPTION
**Pull request by julie-git**
Added styling to customer login, customer reg, handyman reg, handyman login, coded handyman login and reg js.

### CODE REVIEW

File: config/config.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "development": {
     "username": "root",
-    "password": "password",
+    "password": "teddy",
```
Comment (chrisp-dev) [inline config/config.json:4]: STOP CHECKING IN THIS PASSWORD PLEASE.

**Review by chrisp-dev — DISMISSED** [review]

File: config/config.json (hunk 1 above)
Comment (chrisp-dev) [reply config/config.json:4]: Your frontend work has nothing to do with changing the configuration for the database, please refrain and listen to me, do not change this.

**Review by julie-git — COMMENTED** [review]
Sorry I changed it in my local JGStyling3 but can't push the changes.  Could you fix this for me.
It's now blocking me from moving forward with work.

File: config/config.json (hunk 1 above)
Comment (julie-git) [reply config/config.json:4]: Ok. then what do you need me to do to approve the merge? 
The status is "changes requested"

Comment (aritse) [reply config/config.json:4]: just use 'password' across all. this is frustrating. thanks.

**Review by aritse — APPROVED** [review]

**Comment by chrisp-dev** [comment]
Thanks.

**Pull request merged**